src/components/elements/input/RadioGroupInput.tsx

```diff
@@ -64,7 +64,7 @@ const RadioGroupInput = ({
   value,
   error,
   warnIfEmptyTreatment,
-  row,
+  row, // Not supported by dynamic forms anymore, but used internally by YesNoRadio. Support is partial (e.g. helper text is not supported)
```

Comment:
I say lets go ahead and remove it entirely.

Reply:
Sounds good, done. I think this looks fine on the table filters, but are you at all concerned about the impact in the form builder? We use this in EnableWhen in a design that I think assumed the horizontal checkbox layout:

![Screenshot 2024-08-07 at 3 42 54 PM](https://github.com/user-attachments/assets/17c121b2-7834-494e-8626-f5079780bc2f)

Reply:
Hmm good point, I think it could be resolved by switching to a dropdown there. Lets just table it for now pedning the next design session around conditionals, which I think is on the docket for next week

Reply:
Sounds good, I'll go ahead and merge, and we can discuss next week.